### PR TITLE
Prepare `@xstate/react` release

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

File: packages/xstate-react/src/fsm.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export function useMachine<
     if (stateMachine !== initialMachine) {
       throw new Error(
         'Machine given to `useMachine` has changed between renders. This is not supported and might lead to unexpected results.\n' +
-          'Please make sure that you pass the same Machine as argument each time.'
+          "Please make sure that you pass the same Machine as argument each time. Keep also that in mind that `useMemo` can't be used for this kind of caching as per https://reactjs.org/docs/hooks-reference.html#usememo"
```
Comment (tamebadger) [inline packages/xstate-react/src/fsm.ts]: I'd recommend wording this as "Also keep in mind that `useMemo` can't be used for this kind of caching as per https://reactjs.org/docs/hooks-reference.html#usememo""

File: packages/xstate-react/src/index.ts [hunk 2]
```diff
@@ -45,7 +45,7 @@ export function useMachine<
     if (machine !== initialMachine) {
       throw new Error(
         'Machine given to `useMachine` has changed between renders. This is not supported and might lead to unexpected results.\n' +
-          'Please make sure that you pass the same Machine as argument each time.'
+          "Please make sure that you pass the same Machine as argument each time. Keep also that in mind that `useMemo` can't be used for this kind of caching as per https://reactjs.org/docs/hooks-reference.html#usememo"
```
Comment (tamebadger) [inline packages/xstate-react/src/index.ts]: same comment as above 🤓

File: packages/xstate-react/src/fsm.ts (hunk 1 above)
Comment (Andarist) [reply packages/xstate-react/src/fsm.ts]: lol, yea - this is some true past midnight writing 😅 my version is totally incorrect

**Comment by davidkpiano** [comment]
@Andarist Is this still needed?

**Pull request closed without merging**